ee/codegen/src/generators/sandbox-file.ts

```diff
@@ -0,0 +1,113 @@
+    inputsFile: Inputs;
+    sandboxInputs: Record<string, unknown>[];
+  }
+}
+
+export class SandboxFile extends BasePersistedFile {
+  private readonly workflowFile: WorkflowFile;
+  private readonly inputsFile: Inputs;
+  private readonly sandboxInputs: Record<string, unknown>[];
+
+  public constructor({
+    workflowContext,
+    workflowFile,
+    inputsFile,
+    sandboxInputs,
+  }: SandboxFile.Args) {
+    super({ workflowContext });
+    this.workflowFile = workflowFile;
+    this.inputsFile = inputsFile;
+    this.sandboxInputs = sandboxInputs;
+  }
+
+  protected getModulePath(): string[] {
+    return [this.workflowContext.moduleName, "sandbox"];
+  }
+
+  protected getFileStatements(): AstNode[] {
+    const sandboxRunnerField = python.field({
+      name: "runner",
+      initializer: python.instantiateClass({
+        classReference: python.reference({
+          name: "SandboxRunner",
+          modulePath:
+            this.workflowContext.sdkModulePathNames.SANDBOX_RUNNER_MODULE_PATH,
+        }),
+        arguments_: [
+          python.methodArgument({
+            name: "workflow",
+            value: python.reference({
+              name: "Workflow",
+              modulePath: this.workflowFile.getModulePath(),
+            }),
+          }),
+          python.methodArgument({
+            name: "inputs",
+            value: python.TypeInstantiation.list(
+              this.sandboxInputs.map((input) => this.getWorkflowInput(input))
+            ),
+          }),
+        ],
+      }),
+    });
+    this.inheritReferences(sandboxRunnerField);
+
+    return [
+      python.codeBlock(`\
+if __name__ != "__main__":
+    raise Exception("This file is not meant to be imported")
+        `),
+      sandboxRunnerField,
+      python.codeBlock("runner.run()"),
+    ];
+  }
+
+  private getWorkflowInput(
+    input: Record<string, unknown>
+  ): python.ClassInstantiation {
+    return python.instantiateClass({
+      classReference: python.reference({
+        name: "Inputs",
+        modulePath: this.inputsFile.getModulePath(),
+      }),
+      arguments_: Object.entries(input).map(([key, value]) =>
+        python.methodArgument({
+          name: key,
+          value: this.getWorkflowInputValue(value),
+        })
+      ),
+    });
+  }
+
+  private getWorkflowInputValue(value: unknown): python.TypeInstantiation {
+    if (typeof value === "string") {
+      return python.TypeInstantiation.str(value);
+    }
+    if (typeof value === "number") {
+      return python.TypeInstantiation.float(value);
+    }
+    if (typeof value === "boolean") {
+      return python.TypeInstantiation.bool(value);
+    }
+    if (Array.isArray(value)) {
+      return python.TypeInstantiation.list(
+        value.map(this.getWorkflowInputValue)
+      );
+    }
+    if (value === null) {
+      return python.TypeInstantiation.none();
+    }
+    return python.TypeInstantiation.dict(JSON.parse(JSON.stringify(value)));
```

Comment:
What is this case here?

I feel like we already have helpers/ ast nodes for generating vellum value primitives if that's what you're trying to do here.

Reply:
Was originally treating `value` as an `any`, but maybe should be treated as a VellumValue and reuse that logic